c/src/ml-api-service-private.h

```diff
@@ -125,6 +150,16 @@ int _ml_service_query_release_internal (ml_service_s *mls);
  */
 const gchar * _ml_service_get_json_string_member (JsonObject *object, const gchar *member_name);
 
+/**
+ * @brief Invoke ml-service event for new data.
+ */
+int _ml_service_invoke_event_new_data (ml_service_s * mls, const char *name, const ml_tensors_data_h data);
+
+/**
+ * @brief Callback for sink node in pipeline description.
```

Comment:
Ditto.

Reply:
This function processes the data received from the sink element of the pipeline and passes it to the _ml_service_invoke_event_new_data function. 
I will modify it with an appropriate briefing. Thank you. :pray:

Reply:
Updated, Thank you.